src/test/java/ua/lviv/iot/algo/part1/lab3/HelicopterTest.java

```diff
@@ -0,0 +1,65 @@
+package ua.lviv.iot.algo.part1.lab3;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class HelicopterTest {
+    private final Helicopter helicopter = new Helicopter(7000, 13000,
+            "Mi", 70,
+            101, "Hip",
+            2000, 3800,
+            2000, 200,
+            800);
+    private final Helicopter helicopter_null = new Helicopter();
+
+    @Test
+    public void testTakeOff() {
+        int newCurrentAltidute = helicopter.getCurrentAltitude() + 100;
+        helicopter.takeOff();
+        assertEquals(newCurrentAltidute, helicopter.getCurrentAltitude());
+    }
+
+    @Test
+    public void testAscend() {
+        int newCurrentAltidute = helicopter.getCurrentAltitude() + 200;
+        helicopter.ascend(200);
+        assertEquals(newCurrentAltidute, helicopter.getCurrentAltitude());
+        // ascend further than maxAltidute
+        helicopter.ascend(20000);
+        assertEquals(helicopter.getMaxAltitude(), helicopter.getCurrentAltitude());
+    }
+
+    @Test
+    public void testDescend() {
+        int newCurrentAltidute = helicopter.getCurrentAltitude() - 200;
+        helicopter.descend(200);
+        assertEquals(newCurrentAltidute, helicopter.getCurrentAltitude());
+        // descend to 0m and lower
+        helicopter.descend(20000);
+        assertEquals(0, helicopter.getCurrentAltitude());
+    }
+
+    @Test
+    public void refuel() {
+        int newCurrentFuel = helicopter.getCurrentFuel() + 200;
+        helicopter.refuel(200);
+        assertEquals(newCurrentFuel, helicopter.getCurrentFuel());
+        // add too more fuel
```

Comment:
Same as ascend.

P. S. Better to throw exception on fuel overflow (more than capacity limit)